### PR TITLE
OpenStack Victoria support

### PR DESCRIPTION
**Pull request by jouvin**
- memcached added to all services
- Glance Ceph support rafactoring (minor site configuration adjustments needed)

### CODE REVIEW

**Comment by jouvin** [comment]
@all these are templates for Victoria release that have been run at IJCLab. Once they are accepted/merged, I will push Wallaby but I can do only one release at a time if I want to keep the base branch for each release consistent (the previous release and not `ussuri`). If you give me the green light, I can merge the templates for each release as I don't expect much review...

**Comment by jouvin** [comment]
Is it possible to merge this PR, if there are no objection. Or do you want me to do it (not really optimal/clean)?

**Comment by jouvin** [comment]
@jrha @Pansanel do you agree merging this PR? Do you want me to do it?

File: features/nova/compute/ceph.pan
Comment (jrha) [inline features/nova/compute/ceph.pan]: The `if` is redundant here:

```suggestion
variable OS_NOVA_ENABLE_RBD_DOWNLOAD ?= (
    is_defined(OS_GLANCE_BACKEND_PARAMS) && 
    is_defined(OS_GLANCE_BACKEND_DEFAULT) &&
    (OS_GLANCE_BACKEND_PARAMS[OS_GLANCE_BACKEND_DEFAULT]['type'] == 'rbd') 
);
```

Comment (jouvin) [reply features/nova/compute/ceph.pan]: Thanks for spotting this!

**Comment by jouvin** [comment]
@jrha should be ready for merging...

**Review by jrha — APPROVED** [review]

**Pull request merged**
